content/doc/book/installing/kubernetes.adoc: elaborate about file system access (#7963)
jenkins-infra/jenkins.io: content/doc/book/installing/kubernetes.adoc

  * Kubernetes automatically uses this policy by default if the `:latest` image tag is specified, or none is specified at all (then the `:latest` image tag is automatically defaulted).
  * For predictable and deterministic updates, use the version-based image tags instead. Search https://hub.docker.com/r/jenkins/jenkins/tags for the ones currently served.
  * Consider an image tag like `lts-jdk21` to avoid surprises when the Jenkins core switches to using a newer JDK (usually they are backwards compatible, but...) or stay with `lts` for indefinite hands-off maintenance approach.

> . The `spec:/strategy:/type: Recreate` requires that Kubernetes first stops the old container instance, then starts a new one a few seconds later, whenever a restart is requested (e.g. to pull in a new Jenkins version).

Reviewer: ```suggestion
. The `spec:/strategy:/type: Recreate` requires that Kubernetes first stops the old container instance, then starts a new one a few seconds later, whenever a restart is requested such as pulling in a new Jenkins version.
```
Small syntax adjustment to replace `e.g.` with `such as`. This is in reference to [the microsoft style guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg) recommendation.
